src/Files.App/App.xaml.cs

```diff
@@ -321,7 +321,7 @@ private async void Window_Closed(object sender, WindowEventArgs args)
 				SaveSessionTabs();
 				MainPageViewModel.AppInstances.ForEach(tabItem => tabItem.Unload());
 				MainPageViewModel.AppInstances.Clear();
-				await Task.Delay(100);
+				await Task.Delay(500);
```

Comment:
This is not directly related to the quick access widget, but I increased the wait time to ensure that open tabs are closed before sleep.